format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -0,0 +1,91 @@
+package org.scalafmt.internal
+
+import scala.meta._
+import scala.meta.Token
+import scala.meta.tokens.Token._
+
+object SyntaxTokens {
+  private implicit class Util[A <: Tree](val tree: A) extends AnyVal {
+    def findAfter(f: Token => Boolean, p: A => Tree): Option[Token] = {
+      val end = p(tree).tokens.last
+      tree.tokens.dropWhile(_ != end).drop(1).find(f)
+    }
+  }
+  implicit class TermAssignSyntax(val tree: Term.Assign) extends AnyVal {
+    def `=`: Token = tree.findAfter(_.is[Equals], _.lhs).get
```

Comment:
Let's use non-symbolic names for these methods and name them the same as the scala.meta.Token classes. In the old days scalameta tokens were actually named 

```scala
Token.`=`
Token.`=>`
...
```

However, this approach has several drawbacks

* hard to pronounce
* bad tooling support
  * breaks syntax highlighting
  * "go to matching curly" in many editors
  * "Go to class" doesn't work in IntelliJ
  * auto-completions suck

We bit the bullet and converted to non-symbolic names even if this mean having annoying names like `KwNew`. Sure maybe "tooling will improve" but I'm not holding my breadth.